tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/SymbolLookupTest.java

```diff
@@ -393,6 +395,19 @@ public Object[][] getOnFailSymbolPositions() {
         };
     }
 
+    @Test
+    public void testSymbolLookupInModuleAlias() {
+        Project project = BCompileUtil.loadProject("test-src/symbol_lookup_with_module_alias_test.bal");
+        SemanticModel model = getDefaultModulesSemanticModel(project);
+        Document srcFile = getDocumentForSingleSource(project);
+        List<Symbol> visibleSymbols = model.visibleSymbols(srcFile, LinePosition.from(20, 4));
+        List<String> expectedModuleSymbols = List.of("ht", "obj");
+        assertTrue(expectedModuleSymbols.stream()
```

Comment:
What are you trying to assert here?

Reply:
Was trying to check if the module alias information `ht` and `obj` was present in the imported module symbols which were obtained from the visible symbols. 
Used a for loop to further simplify this. 
https://github.com/ballerina-platform/ballerina-lang/pull/38093/commits/79fcbda53a4f011403f7a3deedabf044c871a97a.

Reply:
Updated this with for-loops in https://github.com/ballerina-platform/ballerina-lang/pull/38093/commits/2c54ba165b3b6ff7d3e45b8139f4e38ebfce0ea2.